extensions/wikia/MyHome/MyHome.i18n.php

```diff
@@ -3389,23 +3389,23 @@
 	'tooltip-pt-myhome' => 'Перейти в личный кабинет',
 	'activityfeed' => 'Лента действий',
 	'wiki-activity-message-wall-messages-count' => '{{PLURAL:$1|одно сообщение|$1 сообщений}}',
-);
+];
 
 /** Rusyn (русиньскый)
  * @author Gazeb
  */
-$messages['rue'] = array(
+$messages['rue'] = [
 	'myhome' => 'Моя домашня сторінка',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Моя домашня сторінка" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)